native/src/utils/getNavigationDeepLinks.ts

```diff
@@ -1,13 +1,23 @@
-import type { Position } from 'geojson'
 import { Platform } from 'react-native'
 
+import { LocationModel } from 'api-client/src'
+
 /**
  * Depending on the platform this function returns the correct deeplink for navigation app
  * @param address{string}: location address
- * @param coordinates{number[]}: latitude,longitude
+ * @param location{number[]}: latitude,longitude
  */
-export const getNavigationDeepLinks = (address: string, coordinates: Position): string =>
-  Platform.select({
-    ios: `maps:${coordinates[1]},${coordinates[0]}?q=${address}`,
-    android: `geo:${coordinates[1]},${coordinates[0]}?q=${address}`
+export const getNavigationDeepLinks = (address: string, location: LocationModel): string | undefined => {
+  if (location.coordinates == null) {
+    return undefined
+  }
+
+  const x = location.coordinates[0]
+  const y = location.coordinates[1]
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  return Platform.select({
+    ios: `maps:${y},${x}?q=${address}`,
+    android: `geo:${y},${x}?q=${address}`
   })!
```

Comment:
You can also replace android with default. Thus, you don't have to add the non-null-assertion. Not sure, whether thats actually better style though.

Reply:
Not sure, but I'd stick with explicit android here I guess

Reply:
@Taggotty @klinzo so which one should we use?